app/views/pages/accessibility.html.slim

```diff
@@ -0,0 +1,57 @@
+- content_for :page_title do
+  - t('page_title.accessibility_statement')
+
+- content_for :heading
+	- t('page_title.accessibility_statement')
+
+= render partial: 'layouts/header'
+
+p This website is run by the Ministry of Justice Digital & Technology team. We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
+ul.list.list-bullet
+  li change colours, contrast levels and fonts
+  li zoom in up to 300% without the text spilling off the screen
+  li navigate most of the web form using just a keyboard
+  li navigate most of the web form using speech recognition software
+  li listen to most of the web form using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+p We’ve also made the website text as simple as possible to understand.
+p AbilityNet has advice on making your device easier to use if you have a disability.
+
+h2.heading-large How accessible this website is
+
+p This website is accessible.
+p We have completed a WCAG 2.1 review internally for this website.
+p We used components from the GOV.UK Design System to build this form. The <a href="https://design-system.service.gov.uk/accessibility/">Design System accessibility guide</a> explains the current state of accessibility for these components.
+p Components listed on the Design System accessibility guide are fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard.
+
+h2.heading-large What to do if you cannot access parts of this website
+p If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:
+ul.list.list-bullet
+	li <a href="https://track-a-query.service.justice.gov.uk/cases/open#feedback_comment">Use the feedback form at the bottom of this website</a>
```

Comment:
IMO it would be better to link to the feedback form on the current page rather than jump to another page from live website, something like  `<a href="#feedback_comment">`